src/Oscar.jl

```diff
@@ -189,11 +189,13 @@ include("exports.jl")
 @alias characteristic_polynomial charpoly  # FIXME
 @alias minimal_polynomial minpoly  # FIXME
 
+
```

Comment:
```suggestion
```
Why?

Reply:
This must have happened by accident. Thank you for spotting this.